lib/tasks/multitenant/tenants.rake

```diff
@@ -20,6 +20,11 @@ namespace :multitenant do
       puts(query.any? ? 'Some of the tenants haven\t been suspended' : 'All the right tenants have been suspended')
     end
 
+    desc 'Fix in the background tenant_id missing in alerts, log entries and backend apis'
+    task :fix_missing_in_background => :environment do |_task, relations|
+      SetTenantIdWorker::BatchEnqueueWorker.perform_later(*relations.to_a)
+    end
+
```

Comment:
I have just a confusion, how you are getting relations here and is that the reason to  move from fixes.rake to tenants.rake

Reply:
I just found `fixes.rake` to be more appropriate. The relations user must pass when calling the task. For example
```
$ rake multitenant:tenants:fix_missing_in_background[backend_apis]
```

Reply:
we are expecting realtion to be this array 
 
` relations = ["backend_apis", "log_entries", "alerts"]`

Reply:
yes, one or more of these